src/openfermion/ops/_fermion_operator.py

```diff
@@ -205,3 +205,75 @@ def is_molecular_term(self):
             if not (particles == spin == 0):
                 return False
         return True
+
+    def freeze_orbitals(self, occupied, unoccupied=[], prune=True):
```

Comment:
Avoid empty list initializers.  They lead to all kinds of problems due to a peculiarity in how they are handled in python.  The preferred method is to initialize to None, then in the body of the function test if unoccupied is None and if so set it to [].  This prevents a lot of accidental bugs from happening.

Reply:
The default value is now None.